README.md

```diff
@@ -43,17 +40,94 @@ Using the command line:
         *msbuild build.proj /t:build;test /p:scope=ResourceManagement\Compute*
 
-### To on-board new libraries
-Follow existing library and create a new folder under "ResourceManagement".
-  - Note: To simplify test discovery, the test folder must be named with ".test" or ".tests"
-  
-If for platform reasons that your library won't use NetCore project system, 3 notes
-  - In your library csproject file, set the msbuild property "AutoRestProjects" to "true"
-  - In your test project files, set both "AutoRestProjects" and "SDKTestProject" to "true"
-  - To simplify test discovery, the test folder must be named with ".tests"
+## To on-board new libraries
+
+### Project Structure
+
+In "src\ResourceManagement", you will find projects for services that have already been implemented
+
+  - Each service contains a project for their generated/customized code
+    - The folder 'Generated' contains the generated code
+    - The folder 'Customizations' contains additions to the generated code - this can include additions to the generated partial classes, or additional classes that augment the SDK or call the generated code
+    - The file 'generate.cmd', used to generate library code for the given service, can also be found in this project
+  - Services also contain a project for their tests
+
+### Branches: AutoRest vs. master
+
+The **AutoRest** branch contains the code generated from AutoRest.
+
+The **master** branch contains the code generated from Hydra/Hyak.
+  - Hydra/Hyak is Azure's legacy code generation technology.
+  - This can still be used to generate client libraries, but the project is not being advanced in favor of AutoRest. Your team should move to AutoRest and Swagger as soon as possible.
+
+### Standard Process
+
+ 1. Create fork of [Azure REST API Specs](https://github.com/azure/azure-rest-api-specs)
+ 2. Create fork of [Azure SDK for .NET](https://github.com/azure/azure-sdk-for-net)
+ 3. Create your Swagger specification for your HTTP API. For more information see 
+ [Introduction to Swagger - The World's Most Popular Framework for APIs](http://swagger.io)
+ 4. Install the latest version of AutoRest and use it to generate your C# client. For more info on getting started with AutoRest, 
+ see the [AutoRest repository](https://github.com/Azure/autorest)
+ 5. Create a branch in your fork of Azure SDK for .NET and add your newly generated code to your project. If you don't have
+ a project in the SDK yet, look at some of the existing projects and build one like the others. 
+ 6. **MANDATORY**: Add or update tests for the newly generated code.
+ 7. Once added to the Azure SDK for .NET, Build your local package using command "msbuild build.proj /t:build;package /p:scope=YourService" 
+ (Note, 'YourService' comes from the sub folder under <sdk-repo-root>\src, for example: "ResourceManagement\Compute")
+ 8. Bump up the package version in YourService.nuget.proj
```

Comment:
>Bump up the package version in YourService.nuget.proj [](start = 4, length = 53)

This is true only for master branch. For Autorest branch they will need to change version in project.json file and also AssemblyInfo.cs file.